### PR TITLE
Added rssreader sample for Windows and MacOS

### PR DESCRIPTION
**Pull request by qmatteoq**

### CODE REVIEW

File: samples/rssreader/macos/Pods/DoubleConversion/LICENSE [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Copyright 2006-2011, the V8 project authors. All rights reserved.
```
Comment (tido64) [inline samples/rssreader/macos/Pods/DoubleConversion/LICENSE:1]: The `Pods/` folder should _not_ be checked in. It is generated by CocoaPods.

**Comment by acoates-ms** [comment]
@qmatteoq 
Can we at least add samples/rssreader/macos/Pods to the gitignore.  
We should not be checking in all those files!

File: samples/rssreader/macos/Pods/DoubleConversion/LICENSE (hunk 1 above)
Comment (qmatteoq) [reply samples/rssreader/macos/Pods/DoubleConversion/LICENSE]: Sorry, I didn't realize the default .gitignore doesn't include the Pods folder for MacOS, only for iOS. It should be fixed now.

File: samples/rssreader/components/ActionBar.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { Button, View, ScrollView, Text, TouchableOpacity, StyleSheet } from 'react-native';
+
+export default class ActionBar extends Component {
```
[the remark this answers is not in the document]
Comment (qmatteoq) [reply samples/rssreader/components/ActionBar.js:4]: I've deleted the component, it was a leftover from a previous iteration. Thanks for the heads up!

File: samples/rssreader/components/ActionBar.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  render() {
+    return (
+      <View style={[ styles.toolbar]}>
```
[the remark this answers is not in the document]
Comment (qmatteoq) [reply samples/rssreader/components/ActionBar.js:7]: Component deleted.

File: samples/rssreader/components/ActionBar.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        <Text style={[styles.title]}>
```
[the remark this answers is not in the document]
Comment (qmatteoq) [reply samples/rssreader/components/ActionBar.js:8]: Component deleted.

File: samples/rssreader/ios/rssreaderTests/rssreaderTests.m [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+  while ([date timeIntervalSinceNow] > 0 && !foundElement && !redboxError) {
+    [[NSRunLoop mainRunLoop] runMode:NSDefaultRunLoopMode beforeDate:[NSDate dateWithTimeIntervalSinceNow:0.1]];
+    [[NSRunLoop mainRunLoop] runMode:NSRunLoopCommonModes beforeDate:[NSDate dateWithTimeIntervalSinceNow:0.1]];
+
+    foundElement = [self findSubviewInView:vc.view matching:^BOOL(UIView *view) {
+      if ([view.accessibilityLabel isEqualToString:TEXT_TO_LOOK_FOR]) {
+        return YES;
+      }
+      return NO;
```
[the remark this answers is not in the document]
Comment (qmatteoq) [reply samples/rssreader/ios/rssreaderTests/rssreaderTests.m:59]: Thanks! I'm committing your suggestion, however this is code automatically included in the React Native standard template. I didn't touch the iOS project while building the sample.

**Comment by qmatteoq** [comment]
> I only had a very quick look at the Android, iOS, and macOS parts. Looks good; I only had a few nitpicks and a couple of questions:
> 
> * It looks like this should also work for Android and iOS? Do you want anyone to test them?
> * Do we need to check in `samples/rssreader/msbuild.binlog`? What does it do?

I gave it a try on Android and "kind of" works. The application isn't really tailored for mobile, since the main goal of this sample was to support desktop (Windows and MacOS). I haven't tried it on iOS since I don't have an iOS device, so any feedback on this platform would be super appreciated.

Regarding msbuild.binlog I deleted it. It's a log file that gets created automatically when you build the project for Windows using the CLI.

**Comment by harinikmsft** [comment]
Thanks @qmatteoq 

@acoates-ms , @tido64  - if one of you can complete the review, we can get this merged in time for BUILD next week.

File: samples/TodosFeed/TodosFeed/windows/TodosFeed/Bundle/index.windows.bundle [hunk 6]
```diff
@@ -0,0 +1,9 @@
+/**
+ * 
+ * Be sure to generate this file using the CLI:
+ *
+ * react-native bundle --platform windows --entry-file index.js 
+ *   --bundle-output windows\TodosFeed\Bundle\index.windows.bundle
```
Comment (asklar) [inline samples/TodosFeed/TodosFeed/windows/TodosFeed/Bundle/index.windows.bundle:6]: this file doesn't need to be checked in

**Pull request merged**